301-availability-set-elastic-storage-ecs/azuredeploy.json

```diff
@@ -0,0 +1,726 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the Administrator of the new VMs"
+      },
+      "defaultValue": "ecsadmin"
```

Comment:
remove this, increases surface area - use GEN-UNIQUE in the param file

Reply:
OK